pandas/core/arrays/arrow/array.py

```diff
@@ -349,7 +349,7 @@ def __getitem__(self, item: PositionalIndexer):
                     pa_dtype = pa.string()
                 else:
                     pa_dtype = self._dtype.pyarrow_dtype
-                return type(self)(pa.chunked_array([], type=pa_dtype))
+                return type(self)(pa.array([], type=pa_dtype))
```

Comment:
A chunked array without chunks should in theory also work, so this might point to something else that is buggy?

Reply:
Looking at the error in https://github.com/pandas-dev/pandas/issues/51734, it might be that the type needs to be specified in the chunked_array() call in `_concat_same_type`

Reply:
This could also solve this, but imo we should rather avoid returning something here that creates these problems. When iterating over a chunked array without chunks you’ll get an empty list, which makes determining the dtype tricky, because we would have to implement upcasting logic when getting more than one object

Reply:
Yep I missed the concat_same_type, makes sense when we only have one type

Reply:
Changed